recipe/meta.yaml

```diff
@@ -1,36 +1,36 @@
 {% set name = "numba" %}
-{% set version = "0.56.4" %}
-{% set sha256 = "ab96b731fb9dee12b404b42b7c1fb82c119352648906a80526afa73658895b73" %}
+{% set version = "0.57.0" %}
 {% set numpy_min_ver = "1.18" %}  # [py<39 and not (aarch64 or s390x or arm64)]
 {% set numpy_min_ver = "1.19" %}  # [py<39 and (aarch64 or s390x or arm64)]
 {% set numpy_min_ver = "1.19" %}  # [py==39 and not arm64]
 {% set numpy_min_ver = "1.21" %}  # [(py==39 and arm64) or py==310]
```

Comment:
Since the minimum numpy run version is 1.21, and 3.11 has numpy 1.23, I would simplify by setting this:
```
{% set numpy_min_ver = "1.21" %}  # [py<311]
{% set numpy_min_ver = "1.23" %}  # [py==311]
```
https://github.com/numba/numba/blob/0.57.0/setup.py#L25

Reply:
done